javalib/src/main/scala/java/util/AbstractSequentialList.scala

```diff
@@ -0,0 +1,42 @@
+package java.util
+
+abstract class AbstractSequentialList[E] protected ()
+    extends AbstractList[E] {
+
+  def get(index: Int): E =
+    listIterator(index).next
```

Comment:
I believe this would throw the wrong kind of exception if you call it as `list.get(list.size)`.

Btw, `next()` with `()`, as usual (there are several occurrences of this in the PR).